src/Evolution/DgSubcell/SubcellOptions.cpp

```diff
@@ -44,14 +43,6 @@ SubcellOptions::SubcellOptions(
   if (not only_dg_block_and_group_names_.has_value()) {
     only_dg_block_ids_ = std::vector<size_t>{};
   }
-  if (finite_difference_derivative_order_.has_value() and
-      not alg::found(std::array<size_t, 5>{{2, 4, 6, 8, 10}},
-                     finite_difference_derivative_order_.value())) {
-    PARSE_ERROR(context,
-                "The finite difference order must be Auto or one of "
-                "2,4,6,8,10, but got "
-                    << finite_difference_derivative_order_.value());
-  }
 }
 
```

Comment:
This is an excellent question! Since it's no longer a `std::optional` it can't have a `None` value. Since it's an `enum` the parsing is actually done where the enum is defined: https://github.com/sxs-collaboration/spectre/blob/84d6a6b9e1d1fbb3f501e515a990927cc3ab2554/src/NumericalAlgorithms/FiniteDifference/DerivativeOrder.cpp#L38-L66